src/layout/dom.ts

```diff
@@ -1,9 +1,14 @@
+import { Priority } from "@clarity-types/core";
 import { Constant, NodeChange, NodeInfo, NodeValue, Source } from "@clarity-types/layout";
 import config from "@src/core/config";
+import { schedule } from "@src/core/task";
 import time from "@src/core/time";
+import discover from "@src/layout/discover";
 import selector from "@src/layout/selector";
 
 let index: number = 1;
+const BLACKLIST_TYPES = ["password", "hidden", "email"];
```

Comment:
Can you document the source of these lists with a reference?

Reply:
Synced offline.